auth-web/devops/cloudbuild-pr.yaml

```diff
@@ -21,7 +21,9 @@ steps:
     env:
       - 'VUE_APP_ADDRESS_COMPLETE_KEY=$_VUE_APP_ADDRESS_COMPLETE_KEY'
       - 'VUE_APP_AUTH_LD_CLIENT_ID=$_VUE_APP_AUTH_LD_CLIENT_ID'
+      - 'VUE_APP_CORP_FORMS_URL=$_VUE_APP_CORP_FORMS_URL'
```

Comment:
If the environment variables is not the secret or password, don't need to set it in this file.

Reply:
Where do we set it then?

PS These values are already in 1Password. This is to fetch the values from there, right?